src/App.jsx

```diff
@@ -23,23 +24,10 @@ export const alex = {
   isMarried: false,
 };
 
+const persons = [misha, olya, alex];
+
 export const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    {persons.map(people => <Person person={people} />)}
```

Comment:
don't forget about `key` by mapping
```suggestion
    {persons.map(people => <Person key={person.name} person={people} />)}
```